bitcoin/src/blockdata/opcodes.rs

```diff
@@ -577,6 +570,12 @@ mod tests {
         assert_eq!(s, "    OP_NOP");
     }
 
+    #[test]
+    fn decode_pushnum() {
+        assert_eq!(OP_PUSHNUM_5.decode_pushnum().expect("pushnum"), 5);
+        assert!(OP_DROP.decode_pushnum().is_none());
+    }
```

Comment:
I think it'd be useful to test things around edges: one-below-min, min, max, one-above-max.

Reply:
Since there's only 16, I covered everything.